test/e2e/testsuites/gcsfuse_integration.go

```diff
@@ -133,6 +140,11 @@ func (t *gcsFuseCSIGCSFuseIntegrationTestSuite) DefineTests(driver storageframew
 			}
 		}
 
+		// GRPC is supported after v2.5.1
+		if !v.AtLeast(version.MustParseSemantic("v2.5.1-gke.0")) && grpcEnabled(driver) {
```

Comment:
we are using the gke builds because these tests are intended to run only with gke managed csi driver?  and the OSS driver head uses gcsfuse master head? Is this correct?

Reply:
I also had this question. When we build for the non-managed driver, we build gcsfuse from master. When we build from master, we assign that build a tag version of 999. This automatically qualifies the non-managed driver to run all the tests. This tag is only really applicable when we run the testsuite against the managed driver.

So yes, the tests run against all non-managed driver versions, and for selected gke managed driver versions.

Reply:
Maybe add a comment so we dont forget

Reply:
Done.